.github/pull_request_template.md

```diff
@@ -4,18 +4,27 @@
 
 Issues Fixed (include github issue #):
 
-Are changes expected to change answers?
-- bit for bit (yes/no)
-- more substantial (yes/no)
-
-Specific changes:
-- changes in parm directory (yes/no)
-- changes in module files (yes/no)
-- new tests added or removed (yes/no)
-
-Testing performed:
-- machines:
-- details:
+Do answers change?
+ - [ ] No. Answers are b4b with previous baselines
```

Comment:
Since we start to use nccmp or other utilities to compare some netcdf files, the results are not bit4bit identical. Maybe we can say: NO. Answers are same with previous baselines when using cmp or nccmp

Reply:
I've update the template to include this.